src/redis/geo.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __GEO_H__
+#define __GEO_H__
+
+#include <stddef.h> /* for size_t */
+
+/* Structures used inside geo.c in order to represent points and array of
+ * points on the earth. */
+typedef struct geoPoint {
```

Comment:
oh no temporal types etc

Reply:
it's for https://redis.io/commands/geoadd/ etc